docs/spec/functional_spec.rst

```diff
@@ -608,3 +640,292 @@ Seal the contract and make it ready for use
+
+
+Send tez from a vault to a contract
+-----------------------------------
+
+Internal. Calls the specified vault's ``%vault_send_tez_to_contract``
+entrypoint, sending the specified amount of ``mutez``. from the vault to the
+provided contract address. Fails if the sender is not the wtez contract.
+
+``call_vault_send_tez_to_contract: (pair address (pair tez address))``
+
++---------------+-----------------------+-------------------------------------------------------------------------+
+| Parameter     |      Field Type       | Description                                                             |
++===============+=======================+=========================================================================+
+| vault_address | address               | The address of the vault                                                |
++---------------+-----------------------+-------------------------------------------------------------------------+
+| amount        | tez                   | The amount of mutez to send from the main contract to the vault         |
++---------------+-----------------------+-------------------------------------------------------------------------+
+| recipient     | address               | The address of the contract which will receive the mutez                |
++---------------+-----------------------+-------------------------------------------------------------------------+
+
+
+Send tez from a vault to another vault
+--------------------------------------
+
+Internal. Calls the specified vault's ``%vault_send_tez_to_vault``
+entrypoint, sending the specified amount of ``mutez``. from the vault to the
+provided vault address. Fails if the sender is not the wtez contract.
+
+``call_vault_send_tez_to_vault: (pair address (pair tez address))``
+
++---------------+-----------------------+-------------------------------------------------------------------------+
+| Parameter     |      Field Type       | Description                                                             |
++===============+=======================+=========================================================================+
+| vault_address | address               | The address of the vault                                                |
++---------------+-----------------------+-------------------------------------------------------------------------+
+| amount        | tez                   | The amount of mutez to send from the main contract to the vault         |
++---------------+-----------------------+-------------------------------------------------------------------------+
+| recipient     | address               | The address of the vault which will receive the mutez                   |
++---------------+-----------------------+-------------------------------------------------------------------------+
+
+Set a vault's optional delegate
+-------------------------------
```

Comment:
```suggestion
Set a vault's delegate
----------------------
```
it's pretty standard, the fact that this is an optional so it should be good like this